data/dictionary/TWPhrasesIT.txt

```diff
@@ -373,3 +373,15 @@ SQL注入攻擊	SQL隱碼攻擊
 默認值	預設值
 點擊	點選
 鼠標	滑鼠
+黑客	駭客
+批量	批次
+操作符	運算子
+拷貝	複製
+克隆	複製
+滾動條	捲軸
+花屏	破圖
+藍屏	藍色畫面
+優化	最佳化
+高清	高畫質
+U盤	隨身碟
+閃退	當機
```

Comment:
I feel this is wrong but I can't suggest a better replacement at this moment.

Reply:
I agree and I think there is no precise word for it in zh-TW, or we need to use a "longer" phrase to describe that.

Reply:
You can remove this line until a better word is found